terraso_backend/apps/story_map/permission_rules.py

```diff
@@ -66,6 +66,32 @@ def allowed_to_save_story_map_membership(user, obj):
     return False
 
 
+@rules.predicate
+def allowed_to_approve_story_map_membership(user, obj):
+    membership = obj.get("membership")
+    is_user_membership = membership.user.email == user.email
+    return is_user_membership
+
+
+@rules.predicate
+def allowed_to_approve_story_map_membership_token(user, obj):
```

Comment:
question: what is the difference between `allowed_to_approve_story_map_membership_token` and `allowed_to_approve_story_map_membership`? Please add some documentation/comments explaining what this method does.